packages/lab/src/components/Flow/stories/Usage.stories.mdx

```diff
@@ -0,0 +1,427 @@
+  description?: string;
+  color?: HvColorAny;
+  icon?: React.ReactNode;
+}
+```
+
+Example:
+
+```typescript
+// Node groups
+type NodeGroups = "assets" | "models";
+
+const nodeGroups = {
+  assets: {
+    label: "Assets",
+    color: "cat3_80",
+    description: "This is my description for assets.",
+    icon: <Heart />,
+  },
+  models: {
+    label: "ML Model",
+    color: "cat1_80",
+    description: "This is my description for models.",
+    icon: <Favorite />,
+  },
+} satisfies HvFlowProps<any, NodeType, NodeGroups>["nodeGroups"];
+```
+
+You can then create the sidebar by using the `HvFlowSidebar` component as a prop of the `HvFlow` component:
+
+```tsx
+<HvFlow
+  nodeGroups={nodeGroups}
+  (...)
+  sidebar={
+    <HvFlowSidebar
+      title="Add Node"
+      description="Please choose within the options below"
+      open={open}
+      onClose={() => setOpen(false)}
+    />
+  }
+
+>
+  (...)
+</HvFlow>
+```
+
+## Layout <a id="layout" />
+
+There a few controls that you can add to your flow to control its layout and some functionality. All of these are components that you
+should pass as children of the `HvFlow` component.
+
+### Background <a id="background" />
+
+To add a background to the flow you should use the `<HvFlowBackground />` component. This is a wrapper around the
+[Background](https://reactflow.dev/docs/api/plugin-components/background/) component of `reactflow` and has the same API.
+
+### Controls <a id="controls" />
+
+To add the controls panel to the flow you should use the `<HvFlowControls />` component. This, again, uses the
+[Controls](https://reactflow.dev/docs/api/plugin-components/controls/) component from `reactflow` but you should check [our own
+documentation](?path=/docs/lab-flow--docs) for the `HvFlowControls` component also.
+
+### Minimap <a id="minimap" />
+
+To add the minimap of the flow, just add the `<HvFlowMinimap />` component as a child of the `HvFlow` component.
+
+## Custom nodes and node types <a id="custom-nodes-and-node-types" />
+
+The `HvFlowNode` component allows you to create your own nodes. It provides a base for your own custom nodes and is not to be used directly as a node on the flow.
+There are a few properties that should be set on your node `meta` property: the label, the list of inputs and outputs and the id of the group it belongs to.
+The parameters are passed as a property of the `HvFlowNode` component.
+
+Example custom node:
+
+```tsx
+import { HvFlowNode } from "@hitachivantara/uikit-react-lab";
+
+export const CustomNode = (props) => {
+  return (
+    <HvFlowNode
+      description="CustomNode description"
+      expanded
+      params={[
+        {
+          id: "myParam",
+          label: "Parameter",
+          type: "select",
+          options: ["option 1", "option 2", "option 3"],
+        },
+      ]}
+      {...props}
+    >
+      <div>Node content</div>
+    </HvFlowNode>
+  );
+};
+
+CustomNode.meta = {
+  label: "CustomNode",
+  groupId: "someNodeGroup",
```

Comment:
Maybe we should continue using the `groupId`'s defined earlier just for consistency and not create new groups.